<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="videoText">
       <property name="text">
        <string>Vidéo :</string>
       </property>
       <property name="buddy">
        <cstring>videoInput</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="videoInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pathText">
       <property name="text">
        <string>Chemin :</string>
       </property>
       <property name="buddy">
        <cstring>pathInput</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pathInput"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="pathSearch">
       <property name="text">
        <string>Choisir...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="dlCommand">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Télécharger</string>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logWindow">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>videoInput</tabstop>
  <tabstop>pathInput</tabstop>
  <tabstop>logWindow</tabstop>
  <tabstop>pathSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>videoInput</sender>
   <signal>editingFinished()</signal>
   <receiver>pathInput</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
